### PR TITLE
JESI-4176

### PR DESCRIPTION
**Pull request by axrs**
[JESI-4176](https://jesims.atlassian.net/browse/JESI-4176)

- [x] Backpack Released

### CODE REVIEW

File: package-dry.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "@io.jesi/parent",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (AndreTheHunter) [inline package-dry.json]: Bump minor when updating managed dependency in parent project
See https://github.com/jesims/docs/blob/master/Versioning.md#bumping
```suggestion
  "version": "0.3.0",
```

File: project.clj [hunk 2]
```diff
@@ -1,14 +1,14 @@
-(defproject io.jesi/parent "4.6.0"
+(defproject io.jesi/parent "4.6.1"
```
Comment (AndreTheHunter) [inline project.clj]:
```suggestion
(defproject io.jesi/parent "4.7.0"
```

File: project.clj [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
   :license {:name         "Eclipse Public License - v 1.0"
             :url          "http://www.eclipse.org/legal/epl-v10.html"
             :distribution :repo
             :comments     "same as Clojure"}
   :min-lein-version "2.9.1"
-  :managed-dependencies [[org.clojure/clojure "1.10.2"]
-                         [thheller/shadow-cljs "2.11.18"]
-                         [io.jesi/backpack "6.3.1"]
+  :managed-dependencies [[org.clojure/clojure "1.10.3"]
+                         [thheller/shadow-cljs "2.11.21"]
```
Comment (AndreTheHunter) [inline project.clj:10]: Check `org.clojure/clojurescript` and other versions below.
Match versions used in shadow-cljs https://github.com/thheller/shadow-cljs/blob/master/project.clj

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
+# project.clj 4.6.1 and package-dry.json 0.2.1
```
Comment (AndreTheHunter) [inline CHANGELOG.md]:
```suggestion
# project.clj 4.7.0 and package-dry.json 0.3.0
```

**Review by AndreTheHunter — CHANGES_REQUESTED** [review]

File: project.clj (hunk 3 above)
Comment (axrs) [reply project.clj:10]: They do.

**Review by AndreTheHunter — APPROVED** [review]

**Pull request merged**